mono/utils/mono-context.h

```diff
@@ -31,7 +31,7 @@
 /*HACK, move this to an eventual mono-signal.c*/
 #if defined( __linux__) || defined(__sun) || defined(__APPLE__) || defined(__NetBSD__) || \
        defined(__FreeBSD__) || defined(__OpenBSD__)
-#ifdef HAVE_SIGACTION
+#if defined(HAVE_SIGACTION) || defined(__APPLE__)  // this is required for the tvos simulator
```

Comment:
How about this as a comment:

// The __APPLE__ check is required for the tvos simulator, which has ucontext_t but not sigaction.